CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contribution Guidelines
+
+Most of eProsima's products follow an open source model.
+As such, we are always listening to the community for issues, feature improvements and suggestions.
+
+
+## Submitting a pull request
+
+If writing an issue does not do it for you because you just know what has to be done and how, and you have the energy
+and time to directly contribute to the project, submitting a pull request is your way to go.
+Please mind that by doing so, you accept to abide to the pull request process followed on this repository, meaning:
```

Comment:
```suggestion
Please mind that by doing so, you accept to abide to the pull request process followed by eProsima, meaning:
```
These rules do not apply only to this repository, but also to all eProsima repositories